database/factories/CharacterFactory.php

```diff
@@ -23,11 +23,38 @@ class CharacterFactory extends Factory
      */
     public function definition()
     {
+        $races = [
+            "Dwarf",
+            "Elf",
+            "Halfling",
+            "Human",
+            "Dragonborn",
+            "Gnome",
+            "Half-Elf",
+            "Half-Orc",
+            "Tiefling",
+        ];
+
+        $classes = [
```

Comment:
Artificer is AL too now

Reply:
You can ignore this actually lol I blanked on the fact that this is for the seeder so doesnt ultimately matter.

Reply:
True, but its not in the SRD, and technically not allowed to be used publically on non WoTC systems — im sure it wouldn't matter though since this factory is really only used for testing and local environments, so ill go ahead and add that :)